.github/workflows/tests.yml

```diff
@@ -36,6 +36,29 @@ jobs:
           path: "*.snap"
           retention-days: 5
 
+  unit-tests:
+    name: Run Microceph Unit tests
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Clear FORWARD firewall rules
+        run: tests/scripts/actionutils.sh cleaript
```

Comment:
Minor comment, but these shouldn't impede the unit tests and could be left in

Reply:
makes sense, will change accordingly.